examples/grpc-krotodc/src/test/kotlin/HelloServiceTest.kt

```diff
@@ -0,0 +1,146 @@
+import com.google.common.base.Stopwatch
+import com.linecorp.armeria.client.grpc.GrpcClients
+import com.linecorp.armeria.server.Server
+import example.armeria.grpc.krotodc.Main
+import example.armeria.grpc.krotodc.krotodc.HelloReply
+import example.armeria.grpc.krotodc.krotodc.HelloRequest
+import example.armeria.grpc.krotodc.krotodc.HelloServiceGrpcKroto.HelloServiceCoroutineStub
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.runBlocking
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.util.concurrent.TimeUnit
+
+class HelloServiceTest {
+
+    @ParameterizedTest
+    @MethodSource("uris")
+    fun reply(uri: String) {
+        runBlocking {
+            val helloService = GrpcClients.newClient(uri, HelloServiceCoroutineStub::class.java)
+            assertThat(helloService.hello(HelloRequest("Armeria")).message)
+                .isEqualTo("Hello, Armeria!")
+        }
+    }
+
+    @ParameterizedTest
+    @MethodSource("uris")
+    fun replyWithDelay(uri: String) {
+        runBlocking {
+            val helloService = GrpcClients.newClient(uri, HelloServiceCoroutineStub::class.java)
+            val reply: HelloReply = helloService.lazyHello(HelloRequest("Armeria"))
```

Comment:
It looks convenient to create an object directly from a constructor without builder methods.